test/arbitrable.js

```diff
@@ -0,0 +1,167 @@
+const { assertRevert } = require('@aragon/court/test/helpers/asserts/assertThrow')
+const { assertBn } = require('@aragon/court/test/helpers/asserts/assertBn')
+const { getEventArgument } = require('@aragon/test-helpers/events')
+const { bn, bigExp } = require('@aragon/court/test/helpers/lib/numbers')
+
+const ERC20 = artifacts.require('ERC20Mock')
+const ArbitratorMock = artifacts.require('ArbitratorMock')
+const PrecedenceCampaignArbitrable = artifacts.require('PrecedenceCampaignArbitrable')
+
+const getRawLog = (receipt, contract, eventName) => {
+  const eventAbi = contract.abi.filter(o => o.type === 'event' && o.name === eventName)[0]
+  const rawLog = receipt.receipt.rawLogs.filter(l => l.topics[0] === eventAbi.signature)[0]
+  return web3.eth.abi.decodeLog(
+    eventAbi.inputs,
+    rawLog.data,
+    rawLog.topics.slice(1)
+  )
+}
+
+contract('Precedence Campaign Arbitrable', ([_, owner, other, submitter1, submitter2]) => {
+  const ERROR_NOT_ALLOWED = 'PCA_NOT_ALLOWED'
+
+  const FEE_AMOUNT = bigExp(1, 16)
+  const SUBSCRIPTION_AMOUNT = bigExp(2, 16)
+  const POSSIBLE_RULINGS = 2
+  const METADATA = '0x1234'
+
+  let arbitrator, arbitrable
+  let disputeId
+
+  beforeEach('Deploy contracts', async () => {
+    const token = await ERC20.new('Test Token', 'TT', 18)
+    arbitrator = await ArbitratorMock.new(token.address, FEE_AMOUNT, SUBSCRIPTION_AMOUNT)
+    arbitrable = await PrecedenceCampaignArbitrable.new(owner, arbitrator.address)
+
+    // make sure Arbitrable has funds
+    await token.mint(arbitrable.address, FEE_AMOUNT.mul(bn(1000)))
```

Comment:
Should we add tests to check that tokens are being moved as expected by the mock Arbitrator?